esmvaltool/diag_scripts/shared/plot/aux_plotting.ncl

```diff
@@ -74,12 +74,12 @@ begin
   scriptname = "diag_scripts/shared/plot/aux_plotting.ncl"
   enter_msg(scriptname, funcname)
 
-  ; Flag if RGB or RGBA is used for colors
-  if (dimsizes(dimsizes(styles@colors)).ne.1) then
-    RGB = True
-  else
-    RGB = False
-  end if
+;  ; Flag if RGB or RGBA is used for colors
+;  if (dimsizes(dimsizes(styles@colors)).ne.1) then
+;    RGB = True
+;  else
+;    RGB = False
+;  end if
```

Comment:
Why is this code commented out? Should it be removed? Does this affect other diagnostics?

Reply:
This has been commented out because it is obsolete code (the flag "RGB" is not used anywhere). I think it is probably best to simply removed these lines, so that's what I just did.